frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -13,6 +13,8 @@ function filemanagerutil.getDefaultDir()
         return "/mnt/us/documents"
     elseif Device:isKobo() then
         return "/mnt/onboard"
+    elseif Device:isCervantes() then
```

Comment:
I'd prefer this to be alphabetical, too. It's just easier to scan for stuff that way. :-)